test/e2e/main_test.go

```diff
@@ -24,60 +24,74 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
-const TestTimeout = 5 * time.Minute
-const SinglePodTimeout = 5 * time.Minute
-const RouteTimeout = 240 * time.Second
-const DefaultPollPeriod = 1 * time.Second
-
-var CPU = getEnvWithDefault("INFINISPAN_CPU", "500m")
-var Memory = getEnvWithDefault("INFINISPAN_MEMORY", "512Mi")
-var Namespace = getEnvWithDefault("TESTING_NAMESPACE", "namespace-for-testing")
-var RunLocalOperator = getEnvWithDefault("RUN_LOCAL_OPERATOR", "true")
-
-var kubernetes = testutil.NewTestKubernetes()
-var cluster = util.NewCluster(kubernetes.Kubernetes)
-
-var DefaultClusterName = "test-node-startup"
-
-var DefaultSpec = ispnv1.Infinispan{
-	TypeMeta: testutil.InfinispanTypeMeta,
-	ObjectMeta: metav1.ObjectMeta{
-		Name: DefaultClusterName,
-	},
-	Spec: ispnv1.InfinispanSpec{
-		Service: ispnv1.InfinispanServiceSpec{
-			Type: ispnv1.ServiceTypeDataGrid,
+const (
+	TestTimeout          = 5 * time.Minute
+	SinglePodTimeout     = 5 * time.Minute
+	RouteTimeout         = 240 * time.Second
+	DefaultPollPeriod    = 1 * time.Second
+
+	OperatorUpgradeStageNone = "NONE"
+	OperatorUpgradeStageFrom = "FROM"
+	OperatorUpgradeStageTo   = "TO"
+)
+
+var (
```

Comment:
This was the case before, but is there any reason why these are vars and not constants? I haven't checked everyone, but it seems in most cases the values are never updated after creation.